device-types/Supermicro/SSG-640P-E1CR24H.yml

```diff
@@ -0,0 +1,45 @@
+---
+manufacturer: Supermicro
+model: SuperServer SSG-640P-E1CR24H
+slug: supermicro-superserver-ssg-640p-e1cr24h
+part_number: SSG-640P-E1CR24H
+u_height: 4
+is_full_depth: true
+weight: 39
+weight_unit: kg
+airflow: front-to-rear
+comments: '[Technical Specification](https://www.supermicro.com/en/products/system/storage/4u/ssg-640p-e1cr24h)'
+console-ports:
+  - name: Serial
+    type: de-9
+power-ports:
+  - name: PSU-1
+    type: iec-60320-c14
+    maximum_draw: 1200
+  - name: PSU-2
+    type: iec-60320-c14
+    maximum_draw: 1200
```

Comment:
Are these PSU's modular? If so, they should be modelled under `module-bays` and the associated PSU modules should be added to the module-types folder if they do not already exist